src/props.md

```diff
@@ -139,26 +139,6 @@ See the [styled-system][responsive] docs for more info.
 />
 ```
 
-## `css` prop
-
-All Rebass components include a `css` prop,
```

Comment:
Thanks! But I think this should probably explain how to use `babel-plugin-styled-components` instead of the previous custom implementation. Otherwise I'm guessing it might be confusion/questions as to why it changed

Reply:
Sounds good @jxnblk, I could use that explanation myself 😅 –– will defer to you as to whether it's easier to piggyback on this PR or close it and start fresh. Thanks for update, and great project :bow: